packages/google-pay-integration/src/google-pay-paypal-commerce/types.ts

```diff
@@ -1,7 +1,60 @@
+export type FundingType = string[];
+export type EnableFundingType = FundingType | string;
+
+/**
+ *
+ * PayPal Commerce SDK
+ *
+ */
+export interface PayPalSDK {
+    Googlepay: () => {
+        config: () => Promise<GooglePayConfig>;
+        confirmOrder: (arg0: {
+            orderId: string;
+            paymentMethodData: ConfirmOrderData;
+        }) => Promise<{ status: string }>;
+        initiatePayerAction: () => void;
+    };
+}
+
 export interface PayPalCommerceHostWindow extends Window {
-    paypal?: PayPalGoogleSdk;
+    paypal?: PayPalSDK;
+    paypalLoadScript?(options: PayPalCommerceScriptParams): Promise<{ paypal: PayPalSDK }>;
```

Comment:
I don't think we need this option due to the new way of loading paypal sdk

Reply:
Removed